content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,16 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug:  dropbox-coffee-chats
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+## Event Details ##
+
++ **Who:** This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
The link is already a different colour from the text, so I don't see a reason to bold it. 

I agree about the period and changing "women-only" to "Cis & trans .... ". Just note that the last few event posts have said "Women and non-binary individuals" and have had a period at the end, so we need to decide on a format at some point.